### DISCUSSION TITLE
Stability of the project

### DISCUSSION THREAD

**Discussion by mcorossigo** [BlueTeck/kanboard_plugin_gantt#10] [General]
Hello @BlueTeck 
the readme says: ⚠ Early development version ⚠

As I don't have (for now) a test deployment, I would like to know your opinion on the current stability of the plugin before trying it.
Is someone using it in a prod environment? 

Thank you

**Comment by TimoStahl** [comment]
Hey,
unfortunately we aren't using Kanboard anymore 😞 Therefore I will not finish this plugin.
I think the plugin is usable, but the calculation of start and end dates is not completed.
Feel free to make a fork and continue the development.
Regards,
Timo
